src/stats/utility.py

```diff
@@ -0,0 +1,26 @@
+"""
+This module provides utility functions for the Calculator package in the stats project.
+"""
+
+from dataclasses import asdict, is_dataclass
+from typing import Any
+
+
+def convert_results(dataclass_instance: Any) -> dict:
```

Comment:
Maybe convert_results_to_dict or convert_to_ui_results ? It's not clear to what we convert the results by the name.